istio-control/istio-discovery/values.yaml

```diff
@@ -137,4 +137,5 @@ telemetry:
   v2:
     # For Null VM case now. If enabled, will set disableMixerHttpReports to true and not define mixerReportServer
     # also enable metadata exchange and stats filter.
-    enabled: false
+    enabled: true
+    sdfilter: true
```

Comment:
Definitely not enable sd filter by default. What does `telemetry.enabled` and `telemetry.v2.enabled` control respectively? I'm trying to figure out what is the right layer for this option... Also let's use full name for Stackdriver (stackdriverFilter).

Reply:
the outmost layer is gated by telemetry.enabled and telemetry.v2.enabled together now, I think it should be ok to put stackdriverfilter option under v2 as it is

Reply:
`filter` is an implementation detail. I would suggest just stackdriver

Reply:
What if I just want stackdriver, no prometheus?

Reply:
Yeah, just talked to Xinnan offline, he is going to add a new option here (stats) to selectively enable stats filter.

Reply:
The "stats filter" is sort of an implementation detail though. A user just knows they want telemetry. Setting `telemetry=enabled,telemetry.stats=disabled` feels weird, since stats is sort of a synonym for telemetry? 

Is the stats filter just prometheus? If so would it make sense to put this behind an option `prometheus`?

Reply:
make sense to me. @mandarjog your call :)

Reply:
you are right, stats is implementation detail, prometheus makes more sense actually.

Reply:
yes, lets call it

```yaml
stackdriver:
   enabled: true
   logging: true
   monitoring: true
   topology: true
 prometheus:
   enabled: true
```

Reply:
@douglas-reid @bianpengyuan ?

Reply:
that lgtm.

Reply:
is stackdriver all enabled?

Reply:
wait. I think the naming here is fine, but I don't think we can enable topology by default.